### DISCUSSION TITLE
how could i share my fooocus access on my local machine

### DISCUSSION THREAD

**Discussion by whatiphmedia** [lllyasviel/Fooocus#1556] [Ideas]
this might sound silly, but i have no clue how to code. I need to be able to open up fooocus and have others acccess. Is this possible?

**Comment by MystiqueWind** [comment]
First, determine your IP within the local network and modify your Run.bat (with notepad, for example):
at the end of the line `.\python_embeded\python.exe -s Fooocus\entry_with_update.py`  add `--listen 192.168.0.100` (replace with your ip)
now, you can access your neural network with your phone or other comp. Just type your ip in browser, like:
192.168.0.100:7865

It would be better to create a separate bat file)
Good luck

**Comment by mashb1t** [comment]
You can also add --share as argument when starting Fooocus to get a gradio.live link, which you then can share.
This usually doesn't require modifications in your network / router settings etc.
